package.json

```diff
@@ -1,15 +1,24 @@
 {
   "name": "style-constraints",
   "version": "0.1.0",
+  "source": [
+    "lib/index.js"
+  ],
+  "main": "./dist/index.js",
+  "module": "dist/index.m.js",
+  "files": [
+    "index.js",
```

Comment:
@peterpme I don't think we'll need this (no root `./index.js` file, even after microbundle) ?